classes/class-loader.php

```diff
@@ -463,7 +464,9 @@ public function enqueue_campaigns_assets() {
 
 		$campaign_theme = $post->theme ?? $post->custom['_campaign_page_template'] ?? null;
 
-		if ( ! is_string( $campaign_theme ) || empty( $campaign_theme ) ) {
+		if ( ! is_string( $campaign_theme ) || empty( $campaign_theme )
+			|| ! in_array( $campaign_theme, PostCampaign::LEGACY_THEMES, true )
+		) {
 			return;
 		}
 
```

Comment:
The `theme_antarctic` handle below is not correct, this should be generic. I'll change it if I'm sure the change can't cause issues.

Reply:
On second thoughts, it's not worth the risk changing as this will be removed soon.